vscode-wpilib/resources/media/main.css

```diff
@@ -0,0 +1,956 @@
+  font-size: var(--vscode-editor-font-size, 13px);
+  line-height: 1.4;
+}
+
+.log-entry {
+  margin-bottom: 4px;
+  padding: 2px 4px;
+  border-radius: 2px;
+  position: relative;
+}
+
+.log-entry:hover {
+  background-color: var(--vscode-list-hoverBackground);
+}
+
+.error-log {
+  border-left: 3px solid var(--vscode-testing-iconFailed, #f44336);
+}
+
+.warning-log {
+  border-left: 3px solid var(--vscode-warningForeground, #ff9800);
+}
+
+.print-log {
+  border-left: 3px solid transparent;
+}
+
+.log-message {
+  display: flex;
+  align-items: flex-start;
+}
+
+.timestamp {
+  color: var(--vscode-descriptionForeground);
+  margin-right: 8px;
+  font-size: 12px;
+  user-select: none;
+}
+
+.message-content {
+  flex: 1;
+  word-break: break-word;
+}
+
+.toggle-button {
+  display: inline-block;
+  width: 12px;
+  height: 12px;
+  margin-right: 6px;
+  cursor: pointer;
+  position: relative;
+  top: 2px;
+}
+
+.toggle-button.collapsed::before {
+  content: '►';
+  font-size: 10px;
+  color: var(--vscode-descriptionForeground);
+}
+
+.toggle-button.expanded::before {
+  content: '▼';
+  font-size: 10px;
+  color: var(--vscode-descriptionForeground);
+}
+
+.error-content {
+
+/* Rules to properly handle error collapsing in RioLog */
+.error-content.collapsed .stack-trace,
+.error-content.collapsed .location-info {
+  display: none;
+}
+  margin-left: 12px;
+}
+}
+  margin-left: 12px;
+}
+
+.stack-trace,
+.location-info {
+  margin-top: 4px;
+  margin-left: 12px;
+  color: var(--vscode-descriptionForeground);
+  font-size: 12px;
+}
+
+/* Handle input styling for number inputs */
+input::-webkit-outer-spin-button,
+input::-webkit-inner-spin-button {
+  -webkit-appearance: none;
+  margin: 0;
+}
+
+input[type=number] {
+  -moz-appearance: textfield;
+}
+
+/* File input styling for the log viewer mode */
+input[type="file"] {
+  display: none;
+}
+
+label.toolbar-button {
+  display: inline-block;
+  cursor: pointer;
+}
+
+/* Project Creator and Import Styles */
+.project-container {
+  display: flex;
+  flex-direction: column;
+  margin: 0 auto;
+  padding: 16px;
+  max-width: 800px;
+}
+
+.project-row {
+  display: flex;
+  align-items: center;
+  margin-bottom: 16px;
+  flex-wrap: wrap;
+  gap: 12px;
+}
+
+.project-label {
+  flex: 0 0 140px;
+  font-weight: 500;
+}
+
+.project-input {
+  flex: 1;
+  min-width: 250px;
+  background-color: var(--vscode-input-background);
+  border: 1px solid var(--vscode-input-border, var(--vscode-focusBorder));
+  color: var(--vscode-input-foreground);
+  padding: 6px 8px;
+  border-radius: 4px;
+}
+
+.project-input.error {
+  border-color: var(--vscode-inputValidation-errorBorder);
+  background-color: var(--vscode-inputValidation-errorBackground);
+}
+
+.project-button {
+  border: none;
+  padding: 6px 12px;
+  text-align: center;
+  color: var(--vscode-button-foreground);
+  background: var(--vscode-button-background);
+  border-radius: 4px;
+  font-size: 12px;
+  cursor: pointer;
+}
```

Comment:
I'd prefer to use the existing button utility styles for a lot of these instead of duplicating a lot of this. You can keep these individual classes for specific styles, but generally vscode-button should give you a lot of what you need.